components/generic/react-table/types/react-table-config.d.ts

```diff
@@ -0,0 +1,133 @@
+import {
+  UseColumnOrderInstanceProps,
```

Comment:
do we need to add all this things? I think is enough if you install the react table types https://www.npmjs.com/package/@types/react-table

Reply:
that package isn't updated for new react-table version

Reply:
so what will it happen ? we would need to keep these definitions updated by ourselves here

Reply:
we need to keep these definitions until @types/react-table isn't updated